Cargo.toml

```diff
@@ -29,35 +29,35 @@ rust-version = "1.80"
 [workspace.dependencies.typespec]
 default-features = false
 path = "sdk/typespec"
-version = "0.2.0"
+version = "0.3.0"
 
 [workspace.dependencies.typespec_client_core]
 default-features = false
 path = "sdk/typespec/typespec_client_core"
-version = "0.1.0"
+version = "0.2.0"
 
 [workspace.dependencies.typespec_macros]
-version = "0.1.0"
+version = "0.2.0"
 path = "sdk/typespec/typespec_macros"
 
 [workspace.dependencies.azure_core]
-version = "0.22.0"
+version = "0.23.0"
 path = "sdk/core/azure_core"
 
 [workspace.dependencies.azure_core_amqp]
-version = "0.1.0"
+version = "0.2.0"
 path = "sdk/core/azure_core_amqp"
 
 [workspace.dependencies.azure_core_test]
-version = "0.1.0"
+# azure_core_test is not published and only ever a dev-dependency
 path = "sdk/core/azure_core_test"
 
 [workspace.dependencies.azure_core_test_macros]
-version = "0.1.0"
+# azure_core_test_macros is not published and only ever a dev-dependency
 path = "sdk/core/azure_core_test_macros"
 
 [workspace.dependencies.azure_identity]
-version = "0.22.0"
+# azure_identity should only ever be in dev-dependencies herein
```

Comment:
`azure_storage_blob` is still taking a workspace dependency on `azure_identity`

Reply:
Yes, in a dev-dependency which is now a path. I don't see the problem. Could you clarify?

Reply:
Oh, wait, as a regular dependency too. That's not allowed, @vincenttran-msft. Let me see if it's actually used in shipping code...